apps/web/src/actions/procedures/index.ts

```diff
@@ -3,11 +3,6 @@ import { getCurrentUser } from '$/services/auth/getCurrentUser'
 import { z } from 'zod'
 import { createServerActionProcedure } from 'zsa'
 
-/**
- * Procedures allow you to add additional context to a set of server actions,
- * such as the userId of the caller.
- * Docs: https://zsa.vercel.app/docs/procedures
- */
```

Comment:
Why? For a new person arriving to this code this can be a helpful contextual resource

Reply:
There's really no reason to add a link to the docs for this third party abstraction vs all the other third party abstractions we use. If the dev wants to understand they can always google the docs and we keep code tidy. Let's reserve comments for code that is hard to understand.